solr/contrib/ltr/src/java/org/apache/solr/ltr/model/MultipleAdditiveTreesModel.java

```diff
@@ -292,10 +291,10 @@ private static float scoreNode(float[] featureVector, RegressionTreeNode regress
         return 0f;
       }
 
-      if (featureVector[regressionTreeNode.featureIndex] <= regressionTreeNode.threshold) {
-        regressionTreeNode = regressionTreeNode.left;
-      } else {
+      if (featureVector[regressionTreeNode.featureIndex] >= regressionTreeNode.threshold) {
```

Comment:
I think we should keep it consistent, isn't <= in XGboost and similar?
This is a pretty big change, and I think it shouldn't be necssary

Reply:
The `==` going left or right is an interesting question! One way to maintain backwards compatibility whilst supporting "go right instead of left" could be via an optional configuration element e.g.

```
- "params" : { "trees" : [ ... ] }
+ "params" : { "trees" : [ ... ], "splitToRight" : true }
```

And if there is a `scoreNode` change then `explainNode` requires a matching change I think.

Reply:
We run into case with a specific XGBoost model where a variable might take on 0 or 1 and the split condition being `1.0`. So, in that case if we go left by default the right path is never evaluated, which results in silently decreased model performance. 

But, agree it's a rather big change, there might be issues with a) other tree models that split to the left b) backwards incompatibility, so @cpoerschke suggestion makes a lot of sense.

Reply:
I think `==` should be always on the left. But I agree with `splitToRight` for backward compatibility.

Reply:
@diegoceccarelli not sure if there are different tree models that either split to left or right when equal. At least in the case of the XGBoost model, it seems to split to the right by default. I'll check if I can add that flag functionality on the PR.